DDSView/ddsview.cpp

```diff
@@ -452,8 +452,8 @@ HRESULT InitDevice(const TexMetadata& mdata)
     vp.TopLeftX = vp.TopLeftY = 0.f;
     vp.Width = static_cast<float>(width);
     vp.Height = static_cast<float>(height);
-    vp.MinDepth = D3D11_MIN_DEPTH;
```

Comment:
MinGW's d3d11 headers are pretty quirky. They are missing the ``D3D11_MIN_DEPTH`` and ``D3D11_MAX_DEPTH`` symbols.